### ISSUE TITLE
Curly braces inside PSCustomObject break syntax highlighting 

### ISSUE THREAD

**Issue by kborowinski** [PowerShell/EditorSyntax#86]
### Environment
```powershell
> code-insiders.cmd -v
1.24.0-insider
d29620f2ba89327f51bb6fb2f05dfdd65e2fcafe

> $psEditor.EditorServicesVersion
Major  Minor  Build  Revision
-----  -----  -----  --------
1      7      0      0

> code-insiders.cmd --list-extensions --show-versions
...
ms-vscode.PowerShell@1.7.0
...

> $PSVersionTable
Name                           Value
----                           -----
PSVersion                      5.1.17134.1
PSEdition                      Desktop
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}
BuildVersion                   10.0.17134.1
CLRVersion                     4.0.30319.42000
WSManStackVersion              3.0
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
```

### Issue Description

Curly braces inside PSCustomObject break syntax highlighting.

*Note: @omniomi  is it possible that this is related to #84?*

#### Screenshots
![image](https://user-images.githubusercontent.com/16168755/39717710-638045c0-5234-11e8-9078-2aac678014e5.png)
### Expected Behavior
All parameters inside PSCustomObject should be highlighted

*Workaround:* Enclose offending line with braces:

![image](https://user-images.githubusercontent.com/16168755/39717803-a32ce070-5234-11e8-9ac1-3d823c811f29.png)

**Comment by omniomi** [comment]
Good catch. I'll typically use `( ... )` as a matter of course but the first example is valid. Fixed in #87

**Comment by kborowinski** [comment]
PR tested and works as advertised!
